presto-product-tests/README.md

```diff
@@ -147,6 +147,8 @@ where [profile](#profile) is one of either:
  installation running on a single Docker container and a single node
  installation of kerberized Presto also running on a single Docker container.
  This profile runs Kerberos without impersonation.
+- **singlenode-ldap** - OpenLDAP server installed on top of the environment
+ in [singlenode](#singlenode) profile.
```

Comment:
After our discussion on Slack, this is what I came up with. Let me know how it looks: `Three single node docker containers, one running OpenLDAP server, another with SSL/TLS certificates installed on top of a single node Presto
installation, and third one with a pseudo-distributed Hadoop installation`